test/test_pyfftw_complex.py

```diff
@@ -188,6 +190,31 @@ def test_planner_flags(self):
         fft, ifft = self.run_validate_fft(a, b, axes, 
                     flags=())
 
+    def test_wisdom_only(self):
+        in_shape = self.input_shapes['small_1d']
+        out_shape = self.output_shapes['small_1d']
+
+        axes=(0,)
+        a, b = self.create_test_arrays(in_shape, out_shape)
+        forget_wisdom()
+        # with no wisdom, an error should be raised with FFTW_WISDOM_ONLY
+        # (NB: wisdom is specific to aligned/unaligned distinction, so we
+        # ensure that the arrays are always treated as unaligned. Internally, 
+        # run_validate_fft copies the input arrays, which can randomly
+        # cause them to be aligned or unaligned. Without forcing unaligned, this
+        # could otherwise make the wisdom invalid for future planning...
+        self.assertRaisesRegex(RuntimeError, 'No FFTW wisdom', 
+                self.run_validate_fft, *(a, b, axes), 
+                **{'flags':('FFTW_ESTIMATE', 'FFTW_WISDOM_ONLY'), 
+                   'force_unaligned_data': True})
+        # now plan the FFT
+        self.run_validate_fft(a, b, axes, flags=('FFTW_ESTIMATE',),
+                force_unaligned_data=True)
```

Comment:
I'm not sure how FFTW handles the FFTW unaligned flag. It's not clear to me that it doesn't still need the data to still be on the correct alignment even if it doesn't use it to construct the relevant plan. I know that the planner is very conservative about what it allows.